.github /ISSUE_TEMPLATE/otel_in_practice.md

```diff
@@ -0,0 +1,31 @@
+---
+name: OTel in Practice
+about: Issue tracking a new episode of OTel in Practice
+title: 'OTel in Practice: <name>'
+labels: OTel in Practice
+assignees: ''
+
+---
+<!-- Please remember to change the title of this issue by replacing <name> with the actual name for the session. This will be the title used in event invites -->
+
+### Date and time for the session
+<!-- Add here the proposed date for the event after it's been agreed -->
+
+TBD
+
+### Description used for events
+<!-- Add here a description that will be used in events generated for the event -->
```

Comment:
<!-- Add here a description that will be used in **events** generated for the **event** -->

I presume you mean "**promotional posts** generated for the event"?